templates/default/engage-nginx-proxy-conf.erb

```diff
@@ -1,6 +1,7 @@
 server {
   listen 80 default_server;
   listen [::]:80 default_server ipv6only=on;
+  directio 1M;
```

Comment:
So if I understand correctly this bypasses OS caching for anything > 1M. And we want this because caching is the responsibility of cloudfront? Is 1M low enough to cover everything?